WordPress/src/main/res/layout/reader_interest_card.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<com.google.android.material.card.MaterialCardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/interest_container"
+    style="@style/ReaderCardView"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/root_layout"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <androidx.recyclerview.widget.RecyclerView
+            android:id="@+id/interests_list"
+            android:layout_marginTop="@dimen/margin_extra_large"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintLeft_toLeftOf="parent"
+            app:layout_constraintRight_toRightOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/interests_you_may_like_header" />
+
+        <org.wordpress.android.widgets.WPTextView
+            android:id="@+id/interests_you_may_like_header"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:fontFamily="serif"
+            android:layout_margin="@dimen/margin_extra_large"
+            android:text="@string/reader_discover_interests_header"
+            android:textAppearance="?attr/textAppearanceHeadline6"
+            android:textColor="?attr/colorOnSurface"
```

Comment:
These style attributes can be set using reader styles xml. Same can be done for styling `reader_interest_item`.

Reply:
oh, nice! I never knew that existed. Will address this 🙏

Reply:
Resolved here 0d668b4